src/test/java/uk/gov/hmcts/reform/cwrdapi/service/JsrValidatorInitializerTest.java

```diff
@@ -1,7 +1,10 @@
 package uk.gov.hmcts.reform.cwrdapi.service;
 
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
```

Comment:
Hi @ishratt Any specific reason for using Junit 5 annotations here? 
I was checking the build report for PR and found that this class(JsrValidatorInitializerTest.java) is missing in the report.
https://build.platform.hmcts.net/job/HMCTS_RD/job/rd-caseworker-ref-api/job/PR-234/23/testReport/uk.gov.hmcts.reform.cwrdapi.service/

Here is a report from another PR 223 where this class is present.
https://build.platform.hmcts.net/job/HMCTS_RD/job/rd-caseworker-ref-api/job/PR-223/10/testReport/uk.gov.hmcts.reform.cwrdapi.service/

I believe that due to Junit5, the test suite has not executed the tests in this class. Is it possible to use Junit4 annotations?

Reply:
Hi @nitishsingla1234 thanks for pointing this out. I've used JUnit 5 as the @ParameterizedTest is specific to JUnit 5. This was suggested by SonarCloud in the code smell report. Using JUnit 4 would have required the whole class to be run with Parametrized.class, however in this class we have a couple of tests which do not require the parametrized tests, and hence will throw errors. If we want to keep it in JUnit 4, then we might have to create a separate class just for a test, which is what I was trying to avoid.

Reply:
Thanks for the explanation Ishrat. In current scenario, our test cases are not being executed at all.Probably we will need to create another class.

Reply:
Sure, I'll change that, thanks

Reply:
The latest code looks good to me. Approving this PR.